test/e2e/inputs/syslog/syslog_input_test.go

```diff
@@ -0,0 +1,142 @@
+// go:build !fluentd
+package syslog
+
+import (
+	"fmt"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	logging "github.com/openshift/cluster-logging-operator/api/logging/v1"
+	"github.com/openshift/cluster-logging-operator/internal/runtime"
+	framework "github.com/openshift/cluster-logging-operator/test/framework/e2e"
+	"github.com/openshift/cluster-logging-operator/test/framework/functional"
+	"github.com/openshift/cluster-logging-operator/test/helpers"
+	testruntime "github.com/openshift/cluster-logging-operator/test/runtime"
+	v1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/resource"
+	"time"
+)
+
+var _ = Describe("[ClusterLogForwarder] Inputs Syslog", func() {
+
+	var (
+		logGenerator    = "log-generator"
+		forwarderName   = "my-log-collector"
+		destinationHost string
+		testNamespace   string
+		socatCmd        string
+		e2e             = framework.NewE2ETestFramework()
+		receiver        *framework.VectorHttpReceiverLogStore
+		err             error
+		forwarder       *logging.ClusterLogForwarder
+		cpuRequest      = "500m"
+		memRequest      = "64Mi"
+		destinationPort = 10514
+
+		caFile  = "/etc/collector/syslog/tls.crt"
+		keyFile = "/etc/collector/syslog/tls.key"
+	)
+
+	Describe("with vector collector", func() {
+		BeforeEach(func() {
+			// init the framework
+			e2e = framework.NewE2ETestFramework()
+			forwarder = testruntime.NewClusterLogForwarder()
+			testNamespace = e2e.CreateTestNamespace()
+			forwarder.Namespace = testNamespace
+			forwarder.Name = forwarderName
+
+			// deploy receiver
+			receiver, err = e2e.DeployHttpReceiver(testNamespace)
+			Expect(err).To(BeNil())
+			sa, err := e2e.BuildAuthorizationFor(testNamespace, forwarderName).
+				AllowClusterRole("collect-infrastructure-logs").
+				Create()
+			Expect(err).To(BeNil())
+			forwarder.Spec.ServiceAccountName = sa.Name
+
+			cl := runtime.NewClusterLogging(testNamespace, forwarderName)
+			cl.Spec = logging.ClusterLoggingSpec{
+				Collection: &logging.CollectionSpec{
+					Type: logging.LogCollectionTypeVector,
+					CollectorSpec: logging.CollectorSpec{
+						Resources: &v1.ResourceRequirements{
+							Requests: v1.ResourceList{
+								v1.ResourceMemory: resource.MustParse(memRequest),
```

Comment:
We will be dropping the ClusterLogging resource in this release.  Do you have need at this time to set resource limits?  If not, a CLF named anything other than openshift-logging/instance will get you a vector deployment.  This would mitigate any changes we need to do when dropping ClusterLogging